nilearn/plotting/find_cuts.py

```diff
@@ -214,15 +218,13 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     # during given input value "n_cuts"
     epsilon = np.finfo(np.float32).eps
     difference = abs(round(n_cuts) - n_cuts)
-    message = ("The number of cuts in the given direction %s must be "
-               "an integer which should be greater than 0 and "
-               "less than or equal to %d. You provided n_cuts=%s " % (
-                   direction, this_shape, n_cuts))
-    if not isinstance(n_cuts, numbers.Number) or n_cuts < epsilon:
+    if round(n_cuts) < 1. or difference > epsilon:
+        message = ("The number of cuts in the given direction %s must be "
+                   "an integer which should be greater than 0 and "
```

Comment:
although equivalent for integers maybe "greater than or equal to 1" is more clear than "greater than 0"